packages/gatsby-admin/README.md

```diff
@@ -1,8 +1,28 @@
 # Gatsby Admin
 
-A visual interface to configure your Gatsby site.
+A visual interface to configure your Gatsby site. Currently alpha testing.
 
-## Architecture
+## Getting started
+
+To enable Admin for your local Gatsby site, run `gatsby develop` with the `GATSBY_EXPERIMENTAL_ENABLE_ADMIN` environment variable enabled:
+
+```sh
+GATSBY_EXPERIMENTAL_ENABLE_ADMIN=true gatsby develop
+```
+
+You can then visit `localhost:8000/___admin` to view Admin for your Gatsby site!
+
+![Gatsby Admin homepage showing a list of installed plugins, as well as a search input to search for plugins to install](https://user-images.githubusercontent.com/7525670/95580804-36df9200-0a38-11eb-80a7-fbd847a13da1.png)
+
+When you select an installed plugin or search for a new one, you will be able to install/uninstall/configure it:
+
+![Gatsby Admin plugin page showing the README of the gatsby-plugin-sitemap, as well as an input field to configure the options for said plugin](https://user-images.githubusercontent.com/7525670/95580764-27f8df80-0a38-11eb-9f26-8a2cbbc4b07d.png)
+
+Let us know what you think by hitting the "Send feedback" button in Admin!
```

Comment:
Do we have an umbrella issue for issues about Admin? Sometimes the "Send feedback" input might be not enough and people will open issues. So directly route them to an issue as an alternative?

Reply:
But but feedback fish... :cry:

Reply:
(yes, good idea!)